powerauth-push-server/src/main/java/io/getlime/push/service/PushMessageSenderService.java

```diff
@@ -125,7 +125,7 @@ else if (platform.equals(PushDeviceEntity.Platform.Android)) {
     }
 
     // Send message to iOS platform
```

Comment:
Please write proper JavaDoc comments here - we need to have documentation for parameters. Also, this refactoring does not make much sense - we cannot just introduce methods with 7 parameters, we need functional refactoring...